### ISSUE TITLE
config value 'safe.directory' was not found 

### ISSUE THREAD

**Issue by jeroen** [r-lib/gert#181]
After upgrading to libgit2 1.5.0 some actions are showing:

```
Error in libgit2::git_repository_open_ext : 
  config value 'safe.directory' was not found
```

For example [here](https://github.com/r-universe-org/control-room/runs/7971197744?check_suite_focus=true).

I think this is a new security feature in libgit2, but it seems a bit overzealous?

**Comment by jeroen** [comment]
Downgraded the static build to libgit2 1.4.2 for now until this is resolved upstream.

**Comment by appetrosyan** [comment]
> I think this is a new security feature in libgit2, but it seems a bit overzealous?

Not really. This only matters on multi-user machines. Making it mandatory, is for lack of a better word, bad design.

**Comment by maelle** [comment]
It seems the static build uses a much higher version of libgit now.
